HousingSearchApi/V2/Controllers/SearchController.cs

```diff
@@ -0,0 +1,43 @@
+using Amazon.Lambda.Core;
+using Hackney.Core.Logging;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using HousingSearchApi.V2.Domain.DTOs;
+using HousingSearchApi.V2.UseCase;
+
+namespace HousingSearchApi.V2.Controllers;
+
+[ApiVersion("1")]
```

Comment:
It's V2 now :grin: 
The swagger page should get a dropdown to select between v1 and v2 once you change this

Reply:
https://github.com/LBHackney-IT/housing-search-api/pull/238/commits/0c7d529eb8b31975b5c47ccc2bbd7e1edb7f642b
Good spot!